src/rps_server_rest.py

```diff
@@ -40,7 +40,7 @@ def make_http_request_to_ai(image):
         "shape": [1, 100, 100, 1]
     })
 
-    URL = "https://pepper.lillbonum.se/predict/hand"
+    URL = "http://host.docker.internal:5000/predict/hand"
```

Comment:
I think its better to keep the url to external server

Reply:
fixed